core/src/toga/style/pack.py

```diff
@@ -792,7 +792,7 @@ def _layout_children(
 
         # Translate RTL into left-origin, which effectively flips start/end item
         # alignment.
-        align_items = self.align_items
+        align_items = node.align_items
```

Comment:
Just noticed I'd used `self.align_items` instead of `node.align_items`, which is inconsistent with the rest of the method's usage.

This makes me wonder, though... `layout`, `_layout_node`, and `_layout_children` (or its row/column versions) are all passed the node as an argument. But none of them are class or static methods, so they also receive `self`. which as far as I can tell is always the same widget. `layout` and `_layout_children` currently only *use* `node`, while `_layout_node` freely mixes both `self` and `node`. Would you have any objection to either a) making them class/static methods and referring only to `node`, or b) keeping them as normal methods but removing the `node` argument? Personally I'd vote for the latter.

(`_debug` could also easily be a class method.)